tests/input/cpp/lambda4.cpp

```diff
@@ -0,0 +1,25 @@
+void LicenseManager::deleteLicenseFile_(LicenseDomain const inLicenseDomain, std::function<void(bool const inSuccess, MVError const& inError)> const& inCompletionHandler) {
+switch (inLicenseDomain) {
+case LicenseDomain::User:
+if (CFErrorRef error{}; !LicenseFile::UserDomain().deleteFromDisk_(&error)) {
+return inCompletionHandler(false, MVError(error));
+}
+return inCompletionHandler(true, MVError());
+case LicenseDomain::Local:
+deleteSharedLicenseFile_([=](bool const inSuccess, MVError const& inError){
+if (inSuccess || inError.isUserCancelledError()) {
+return inCompletionHandler(inSuccess, inError);
+}
+if (CFErrorRef error{}; !LicenseFile::LocalDomain().deleteFromDisk_(&error)) {
+return inCompletionHandler(true, inError);
+}
+inCompletionHandler(false, inError);
+});
+auto const x = [=](bool const inSuccess, MVError const& inError){
+if (inSuccess || inError.isUserCancelledError()) {
+return inCompletionHandler(inSuccess, inError);
+}
+inCompletionHandler(false, inError);
+};
+}
+}
```

Comment:
Please reformat this input file with an empty config, this way I can see what your option is doing without all the stuff that is happening by default.